Frends.JSON.QuerySingle/Frends.JSON.QuerySingle/Definitions/Result.cs

```diff
@@ -15,7 +15,7 @@ public class Result
     /// Result data.
     /// </summary>
     /// <example>{{ "Name": "Foo", "Products": [{ "Name": "Bar", "Price": 1 }]}}</example>
-    public object Data { get; private set; }
+    public dynamic Data { get; private set; }
```

Comment:
Looks good but could you update this change to QuerySingle's XML result doc?
/// <returns>Object { bool Success, object Data }</returns>